packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,7 +146,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root' className={ css('ms-Dropdown-container') }>
         { label && (
-          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
+          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } htmlFor={ id } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
This label is being added dropdowns to prevent the one on the people picker example page from mistakenly being read instead of the people picker elements, so the first list item shouldn't be added here.

I'll file a separate issue about the first item being skipped by the reader.